tests/fv3_conf/cpld_control_run.IN

```diff
@@ -33,7 +33,7 @@ else
        FV3_IC=@[INPUTDATA_ROOT]/${FV3_DIR}/INPUT_L127_mx${OCNRES}
      fi
      MOM_IC=@[INPUTDATA_ROOT]/MOM6_IC
-     ICE_IC=@[INPUTDATA_ROOT]/CICE_IC/${OCNRES}
+     ICE_IC=@[INPUTDATA_ROOT]/CICE_IC
```

Comment:
Will this directory update impact current icee initial condition in the current tests?

Reply:
No, because below I changed the non-warmstart path for the ice restarts. The actual file location hasn't changed, just how we construct the path name. All the tests were B4b, except for the SFS test.